lib/modules/ai_bot/bot.rb

```diff
@@ -173,6 +168,8 @@ def reply_to(
           end
         end
 
+        puts reply
+
```

Comment:
```suggestion
```

I'm guessing this should be removed.

Reply:
ooops I knew I forgot something, great catch!